src/sass/file-preview-card/file-preview-card.scss

```diff
@@ -0,0 +1,143 @@
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.file-preview-card {
+    @include background-color-token(
+        file-preview-card-background-color,
+        color-background-secondary
+    );
+
+    @include border-radius-token(
+        file-preview-card-border-radius,
+        border-radius-100
+    );
+
+    // Safari support in v15+ reviewed with team
+    aspect-ratio: 1/1;
+    box-sizing: border-box;
+    display: grid;
+    grid-template-areas: " . . menu-action " " . center-action . " " label . . ";
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    height: 100%;
+    max-width: 396px;
+    min-width: 120px;
+    width: 100%;
+}
+
+.file-preview-card__center {
+    align-items: center;
+    display: flex;
+    grid-area: 1 / 1 / 4 / 4;
+    justify-content: center;
+}
+
+.file-preview-card__center img {
+    @include border-radius-token(
+        file-preview-card-border-radius,
+        border-radius-100
+    );
+
+    height: 100%;
+    object-fit: contain;
+    width: 100%;
+}
+
+.file-preview-card__center .inline-notice--attention {
+    @include color-token(
+        file-preview-card-attention-color,
+        color-foreground-attention
+    );
+
+    font-size: var(--font-size-12);
+    line-height: var(--spacing-200, 16px);
+}
+
+.file-preview-card__center--overflow {
+    opacity: 0.5;
+}
+
+.file-preview-card .file-preview-card__menu-action {
+    grid-area: menu-action;
+    justify-self: end;
+}
+
+.file-preview-card .file-preview-card__menu-action .icon-btn {
+    @include background-color-token(
+        file-preview-card-menu-action-background-color,
+        color-background-primary
+    );
+
+    height: 32px;
+    margin: var(--spacing-100, 8px);
+    width: 32px;
+}
+
+.file-preview-card__label {
+    align-self: end;
+    grid-area: label;
+}
+
+.file-preview-card__label .icon-btn {
+    @include background-color-token(
+        file-preview-card-label-background-color,
+        color-background-primary
+    );
+
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    gap: var(--spacing-50, 4px);
+    height: 20px;
+    margin: var(--spacing-100, 8px);
+    width: 55px;
+}
+
+.file-preview-card__label-text {
+    @include color-token(
+        file-preview-card-label-text-color,
+        color-foreground-primary
+    );
+
+    font-size: var(--font-size-12);
+    font-weight: 400;
+}
+
+.file-preview-card .file-preview-card__center-action {
+    align-self: center;
+    grid-area: center-action;
+    justify-self: center;
+    z-index: 1;
+}
+
+.file-preview-card .file-preview-card__center-action .icon-btn {
+    @include color-token(
+        file-preview-card-center-action-color,
+        color-foreground-on-strong
+    );
+
+    background-color: rgba(var(--color-background-strong-rgb), 0.5);
```

Comment:
@agliga how should we use the `background-color-token` mixin for something like this that is manual?

Reply:
I added a new mixin `background-color-rgba-token`